<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfacing</class>
 <widget class="QMainWindow" name="interfacing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>interfacing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>771</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Terminal </string>
     </attribute>
     <widget class="QPlainTextEdit" name="txt_terminal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>731</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Graph Plot</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Table</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-1</y>
      <width>761</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>SERIAL UART SETTING</string>
    </property>
    <widget class="QCheckBox" name="chkLocalEcho">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Local Echo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BaudRate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_Baud">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>38400</string>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PortName</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_Port">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM2</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_Open">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>52</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>StopBit</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_parity">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>No</string>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmb_StopBit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmb_FlowControl">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>No</string>
     </property>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hardware</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Software</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmb_DataBits">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>8</string>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FlowControl</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DataBits</string>
     </property>
    </widget>
    <zorder>chkLocalEcho</zorder>
    <zorder>label_4</zorder>
    <zorder>cmb_Baud</zorder>
    <zorder>label_5</zorder>
    <zorder>cmb_Port</zorder>
    <zorder>btn_Open</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>tabWidget</zorder>
    <zorder>cmb_parity</zorder>
    <zorder>cmb_StopBit</zorder>
    <zorder>cmb_FlowControl</zorder>
    <zorder>cmb_DataBits</zorder>
    <zorder>label_16</zorder>
    <zorder>label_17</zorder>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>toolBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
